packages/node/src/integrations/anr/index.ts

```diff
@@ -100,6 +101,13 @@ type AnrReturn = (options?: Partial<AnrIntegrationOptions>) => Integration & Anr
 
 export const anrIntegration = defineIntegration(_anrIntegration) as AnrReturn;
 
+function onModuleLoad(callback: () => void): void {
+  // eslint-disable-next-line deprecation/deprecation
+  diagnosticsChannel.channel('module.require.end').subscribe(() => callback());
+  // eslint-disable-next-line deprecation/deprecation
+  diagnosticsChannel.channel('module.import.asyncEnd').subscribe(() => callback());
```

Comment:
Why are these deprecated?

Reply:
There's a new API in `diagnostics_channel` to subscribe to a channel but we need to support older Node versions where this doesn't exist.